packages/Responsive/Responsive.md

```diff
@@ -65,9 +67,63 @@ initialState = {
 * [Tooltip component tests](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/packages/Tooltip/__tests__/Tooltip.spec.jsx#L56-L102) and its corresponding [JSX](https://github.com/telusdigital/tds/blob/b2108d1074383ba887c5b87a2c3055799937fcd3/packages/Tooltip/Tooltip.jsx#L81-L108) rendering a particular bubble only when the media query matches
+
+#### SASS Media Queries
+
+SASS-based media queries are available, and are recommended for static sites. This style of responsiveness provides a few benefits like:
+* Prevents sudden scaling when server-side rendered components first appear
+* Prevents the need to wrap Responsive around your component
+  * This in turn removes the requirement for separate CSS classes for different screen sizes that your application must programmatically switch between.
+
+However, using this implementation means losing the ability to programmatically change what is rendered through JSX on resize, so keep the design of your component in mind.
+
+To make use of SASS media queries through the Responsive component, first declare the following import at the top of your .scss file.
+
+```scss
+@import '~@tds/core-responsive/responsive';
+```
+
+Once this is done, you may leverage [**sass-mq**](https://github.com/sass-mq/sass-mq) style media queries to add responsive behaviour to your component.
+
+```scss
+.someClass {
+  @include mq($from: sm, $until: lg) {
+    width: 500px;
+  }
+  @include mq($from: xl) {
+    width: 900px;
+  }
+}
+
+.someOtherClass {
+  @include mq($until: md, $and: '(orientation: landscape)') {
+    width: 100%;
+    height: 100%;
+  }
+}
+```
+
+#### MQ Parameters
+
+| Parameter     |Values | Effect                                                                                                                         |
+| ------------- |-|-----------------------------------------------------------------------------------------------------------------------------------|
+| `$from`       | `sm`, `md`, `lg`, `xl` |The CSS rules contained within will start at the defined point.                                                                        |
+| `$until`      | `sm`, `md`, `lg`, `xl` |The CSS rules contained within will end at the defined point.                                                                          |
+| `$and`        | Standard CSS Media Query Arguments |Allows for custom arguments like orientation or DPI specifications using the standard CSS syntax. These are encased in quotation marks.|
+
+#### MQ Default Breakpoints
+
+SASS-based media queries will accept the same pre-defined breakpoints as the React component. `sm`, `md`, `lg` and `xl` are all available and break at the same screen widths.
```

Comment:
Change `SASS` to `SCSS`

Reply:
Actually, I think you can just say "Sass". Don't use all caps. https://sass-lang.com/

Reply:
So we're clear, what is the best time to say Sass and the best time to say SCSS? My bad on the all caps, but I would personally prefer to use "Sass" everywhere because that's the name of the processor.

Reply:
Yep, say Sass everywhere. No need to ever say SCSS unless you are actually writing a file extension such as `my-styles.scss`